### DISCUSSION TITLE
TMDB Lists only returning 20 results | How to return larger custom lists?

### DISCUSSION THREAD

**Discussion by DaydreamHero** [meisnate12/Plex-Meta-Manager#1685] [Q&A] [answered]
I am sure I'm not the first person to have this issue, but I couldn't find an answer in the Q&A, guessing I am searching for the wrong thing; I apologize if this is duplicative.
I am using movie.yml which includes the following:
```
  Academy Awards - Best Picture Winner:
    tmdb_list: 8275610
    collection_order: custom
    sync_mode: sync
```
There are 95 movies on that list and my Plex should match 100 files (editions) [as I have a manual list I am looking to replace]; but my return through PMM is 20, the first 20 that appear in TMDB. Looking through the TMDB (https://www.themoviedb.org/talk/5fbc2dd217505100420e0a60?language=en-CA), there are a number of notes that you can only get 20 results through the API. I have tried placing the following above collection_order above:
`    page: 3`
,etc. but this doesn't seem to be the answer. How are others pulling their custom lists? Should I be using something other than TMDB (would suck to start over from what I've done, but c'est la vie.

**Comment by chazlarson** [comment]
I'm not aware of a way around this.  You may need to create an MDB or trakt list.

**Reply by DaydreamHero** [reply]
@chazlarson , when you say create an MDB list, is that IMDB or is that yet another source? I was figuring I might have to go IMDB, but was hoping to salvage my previous efforts. From your experience, any advantage/disadvantage between those options? (I feel like TMDB worked quicker than others for the way I add stuff, but maybe that's just recency bias.) Thx

**Reply by chazlarson** [reply]
mdblist.com

**Comment by meisnate12** [comment, marked as answer]
so this is actually a bug with TMDbAPIs the API changed from returning the whole list to return a paginated list ill adjust and update shortly

**Reply by DaydreamHero** [reply]
Well that is good news, that's odd since their Q&A definitely led me to believe that the 20 items was expected. I'll stand by before I move everything somewhere else ... I'm testing just doing everything in IMDB first since it returns a lot more than I can get anywhere else. Thanks for the response (and for the building block)!

**Reply by braydin72** [reply]
I came here to ask the same thing. Any update?
